charts/argo-cd/templates/argocd-server/clusterrole.yaml

```diff
@@ -27,7 +27,7 @@ rules:
       - pods/log
     verbs:
       - get
-  {{- if eq (index .Values.server.config "exec.enabled") "true" }}
+  {{- if eq (index .Values.configs.cm "exec.enabled") "true" }}
```

Comment:
@mkilchhofer Why was this added into Cluster admin role for Argo CD? Do we expect that admin shouldn't be able to check logs?

Reply:
It will be turned on by in future releases. Argoproj core team decided to disable this by default to not break users setups.
And also in my current position at Swiss Post, our security engineering decided to not enabled that capability on prod clusters.

Reply:
I see - it makes sense from this perspective.